pkg/controllers/inventory/inventory.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2022 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package inventory
```

Comment:
move this under cloud/inventory.go.. We dont need separate dir. We can reorg the dirs later.

Reply:
We can't move it to pkg/controller/cloud as inventory.go is in package inventory. I tried and compiler complained.

Reply:
package need not be inventory. It will be under cloud package.